.github/workflows/env-setup-init-redis-cluster-redis.yml

```diff
@@ -0,0 +1,37 @@
+name: Env Setup - Initialize Redis Cluster in Openshift
+run-name: Env Setup - Initialize Redis Cluster in Openshift from ${{ inputs.gitRef }} to ${{ inputs.environment }}
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: environment
+      gitRef:
+        description: "Build Ref"
+        required: true
+        default: ""
+      redisPort:
+        description: "Redis Port"
+        required: false
+        default: 6379
+
+jobs:
+  recoverRedis:
+    name: Deploy Redis in Openshift
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
+    steps:
+      - name: Checkout Target Branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.gitRef }}
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=${{ vars.OPENSHIFT_CLUSTER_URL }}
+      - name: Init Redis
+        working-directory: "./devops/"
+        run: |
+          make init-redis-cluster NAMESPACE=${{ secrets.OPENSHIFT_ENV_NAMESPACE }} REDIS_PORT=${{ inputs.redisPort }}
```

Comment:
I found an issue here as it will prompt a question. I'm checking how to automatically answer 'yes' to the question.

Reply:
It seems that `echo "yes" |  make command" may do the trick. Another option is the command `yes`.

Reply:
Would you like to have this PR go through to execute some additional testing once merged?

Reply:
Sure, I'll test it after merging. It's just that for this one there's a prompt for the user to type "yes" to continue:
![image](https://github.com/bcgov/SIMS/assets/78114138/c33ea76a-9bf2-4376-b7a2-00e5dca77727)
I did a test and this works:
![image](https://github.com/bcgov/SIMS/assets/78114138/7027681d-b25e-41c0-8c1f-ba02500561c1)